packages/valory/agents/hello_world/aea-config.yaml

```diff
@@ -63,9 +64,19 @@ default_connection: null
 public_id: valory/hello_world_abci:0.1.0
 type: skill
 models:
+  benchmark_tool:
+    args:
+      log_dir: ${str:/benchmarks}
   params:
     args:
-      hello_world_message: ${MY_HELLO_WORLD_MESSAGE:str:HELLO_WORLD!}
+      hello_world_message: ${str:HELLO_WORLD!}
+      service_registry_address: null
+      share_tm_config_on_startup: false
+      on_chain_service_id: null
```

Comment:
Shouldn't these parameters follow the ${<type>:<default_value>} format?

Reply:
Fixed on 868df67ff2082675964b0fd6fae3ede68cb17250.